app/assets/stylesheets/application.css

```diff
@@ -275,14 +278,17 @@ button:hover {
 
 .create_entity,
 input[type="submit"] {
-  background-color: #3a7f0a;
-  padding: 18px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background-color: #7fd543;
+  padding: 28px;
   text-decoration: none;
   color: white;
   position: absolute;
   bottom: 0;
   left: 0;
-  border: none;
+  border: 2px solid white;
   right: 0;
 }
```

Comment:
Great job @Mutai-Gilbert 🤝

- [ ] Great job on centering this button just as the previous reviewer suggested. However, you forgot to add the inner white border just like the design appears in the design guidelines.
- Your design
![your-button](https://github.com/Mutai-Gilbert/budget-app/assets/63917725/f823eeb2-771d-43e2-aaea-67f963a905f3)
- Given design
![given-button](https://github.com/Mutai-Gilbert/budget-app/assets/63917725/32dbba09-f54b-4f6b-9135-6b66d4c830cb)